web/public/js/controllers/manage-dojo-users-controller.js

```diff
@@ -60,11 +62,243 @@ function cdManageDojoUsersCtrl($scope, $state, $q, cdDojoService, alertService,
     $scope.loadPage(false);
   };
 
+  $scope.selectedItems = [];
+
+  $scope.showActionBar = false;
+
+  $scope.$watch('selectedItems', function (newValue) {
```

Comment:
How does badgeSelection behave when switching page? it is cleared when you select another user, am i right ?
So that if you selected a user before, then switch page, the bottom bar will still show up. Is it by design ? (multi-page)